pkg/epforwarder/epforwarder.go

```diff
@@ -312,15 +312,20 @@ func newHTTPPassthroughPipeline(desc passthroughPipelineDesc, destinationsContex
 		encoder = sender.NewGzipContentEncoding(endpoints.Main.CompressionLevel)
 	}
 
-	strategy := sender.NewBatchStrategy(inputChan,
-		senderInput,
-		make(chan struct{}),
-		sender.ArraySerializer,
-		endpoints.BatchWait,
-		endpoints.BatchMaxSize,
-		endpoints.BatchMaxContentSize,
-		desc.eventType,
-		encoder)
+	var strategy sender.Strategy
+	if desc.contentType == http.ProtobufContentType {
```

Comment:
If the main endpoint is configured with `UseCompression = true` (so we use the gzip encoder) and the content type is protobuf (so we use stream strategy), will the intake accept those payloads? 

I'm not sure - so I wanted to check if this was considered, or if it's even possible to configure the `epforwader` this way.

Reply:
Based on my testing, yes. The intake seems to accept both gzip and unzipped protobuf payloads - I was able to upload gzipped container images and sboms and see it in the app but I would appreciate if someone else could confirm.